sklearn/calibration.py

```diff
@@ -270,10 +290,6 @@ def fit(self, X, y, sample_weight=None):
             )
             self.calibrated_classifiers_.append(calibrated_classifier)
         else:
-            X, y = self._validate_data(
-                X, y, accept_sparse=['csc', 'csr', 'coo'],
-                force_all_finite=False, allow_nd=True
-            )
```

Comment:
For consistency I prefer, to never validate the data in the meta estimator and use `_safe_indexing` instead. I am not sure if this is right or not.